docs/experiments-cbqa.md

```diff
@@ -0,0 +1,45 @@
+# PyGaggle: Dense Passage Retrieval (DPR) Baselines
+
+This page contains instructions for running the Closed Book Question Answering.
+
+First install Pygaggle
+
+```
+git clone --recursive https://github.com/castorini/pygaggle.git
+```
+
+Then
+
+```
+pip install pygaggle/
+```
+
+## Models
+
++ Dense Passage Retrieval for Open-Domain Question Answering [(Karpukhin et al., 2020)](https://arxiv.org/pdf/2004.04906.pdf)
+
+## Natural Questions (NQ)
+We first download the retrieval results from Pyserini:
+```bash
+wget https://www.dropbox.com/s/flby7nmthaqbzxo/run.dpr.nq-test.single.bf.json -P data
+```
+
+Then we run the inference and evaluation:
+```bash
+python -um pygaggle.run.evaluate_closed_book_question_answering --data data/run.dpr.nq-test.single.bf.json \
+                                                                --model-name google/t5-large-ssm-nq \
+                                                                --device cuda:0
+```
+
+The following output will be visible after it has finished:
+```
+Exact Match Accuracy: 28.89196675900277
```

Comment:
https://huggingface.co/google/t5-3b-ssm-nq says the score for large is a bit higher? Can you try to quickly see why it might be?